SU-HSDO/suhumsci
# STN-559 : Refine Hero Height

# READY FOR REVIEW

## Summary
1. Reduce the height of hero elements in hero region and components (main content of page).
2. Fixes issue where text overflows hero area.

Reference: [JIRA STN-559](https://sparkbox.atlassian.net/browse/STN-559)

Acceptance Criteria:
* Decrease the height of the hero by approximately 200px.
* Text area flexes with content from a minimum height and button styling
* Padding between elements should not change

## Need Review By (Date)
9/17/2020

## Urgency
medium

## Steps to Test
1. In the CLI, run `npm run test` and confirm all our tests pass.
2. Go to http://sparkbox-sandbox.suhumsci.loc/qa/hero-height-refinements to review hero refinements. You will need to sync with the dev or staging environments to grab the data to test this page.
Review the full-width hero at the top of the page AND the hero located within the main content of the page.
- [x] Colorful theme hero on mobile
- [x] Colorful theme hero on desktop
- [x] Traditional theme hero on mobile
- [x] Traditional theme hero on desktop
3. Because the hero and carousel share some common markup, confirm the carousel in the colorful theme displays as expected. http://sparkbox-sandbox.suhumsci.loc/qa/carousel-hero-image Note: There is a card in the backlog to address traditional carousel refinements.
